editoast/src/core/stdcm.rs

```diff
@@ -56,6 +57,8 @@ pub struct STDCMRequest {
     pub margin: Option<MarginValue>,
     /// List of planned work schedules
     pub work_schedules: Vec<LightWorkSchedule>,
+    // List of applicable temporary speed limits between the train departure and arrival
+    pub temporary_speed_limits: Vec<LightTemporarySpeedLimit>,
```

Comment:
Seeing this, I would expect the code on `core` to change to, notably `STDCCMRequestV2.kt` at the very least.

Reply:
If if does not break core I would rather update the core part in another branch that updates STDCM with the temporary speed limits. Are you fine with me resolving that conversation ?